src/01-Static-HTML/02-Static-HTML-unscaffolded.purs

```diff
@@ -0,0 +1,62 @@
+module StaticHTML.StaticHTMLUnscaffolded where
+
+module Main where
+
+import Prelude 
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Data.Const (Const)
+
+import Halogen as H
+import Halogen.Aff as HA
+import Halogen.VDom.Driver (runUI)
+import Halogen.HTML as HH
+import Halogen.HTML (HTML)
+
+type Query = Const Unit
+type Input = Unit
+type State = Unit
+type Output = Void
+data Action = Unit
+
+-- | Shows how to use Halogen VDOM DSL to render HTML without properties or CSS
+staticHtml :: StaticHTML
+staticHtml =
+  HH.div_
+    -- The 'div' tag takes an Array of children
+    [ HH.div_
+      [ HH.span_
+        -- as does the `span` tag
+        [ HH.text "This is text in a span!" ]
+      ]
+    , HH.button_
+      [ HH.text "You can click me, but I don't do anything." ]
+    ]
+
+render :: State -> HTML (H.ComponentSlot HTML () Aff Action) Action
```

Comment:
Yeah, it makes much sense why things are hidden like this more clearly now... and maybe rather than introduce these unscaffolded versions like I originally thought was a good idea, we might want to treat opening up the types as its own "lesson" later on, but reverting back to the simple components like in the above code, to make sure we don't have too much complexity all at once. Needs more thinking :)

I really like that this (your) halogen tute/doc starts with an overview of the fact that the single component type has `state`, `action`, `input`, `message` and `query` types, and that you explain that you're going to hide detail at first. That lets the reader defer worrying about that, and is excellent.

I suppose in deferring that, I was also setting up an expectation that each little bit of it would be explained in intricate detail later on, but somewhere between points 2 and 3 in the below, that didn't happen for me, for some reason:

> 1. Render static HTML and CSS (components without state) via Halogen VDOM's DSL
> 2. Render dynamic HTML (components with state and simple event handling)
>   introduce the state type and action type
>   briefly overview monads and how they work
> 3. Overview parent and child relationships among components
>   introduce the input, message, and query types
>   overview the action type again
> 4. Overview lifecycle handlers
> ...

So maybe I should flesh out a bit more of an overview of what I mean by "not skipping steps" or whatever it is I'm trying to say here :) (I really hope I'm not being careless here and offending you, because you've done such an amazingly thorough job it's *super* awesome).